components/admin/questDetails.tsx

```diff
@@ -238,6 +242,11 @@ const AdminQuestDetails: FunctionComponent<QuestDetailsProps> = ({
             <p>Go To Quest</p>
           </Button>
         </div>
+        {isEdit && quest.boosts?.length > 0 && (
```

Comment:
Here you should do isEdit ? quest.boosts.map(boost => <div ...></div>

Reply:
Got it, Sir, something like this?
![image](https://github.com/user-attachments/assets/a7d91fb0-8c29-40db-b6eb-9cbacd2e87ef)